includes/admin/meta-boxes/views/html-variation-admin.php

```diff
@@ -48,6 +48,17 @@
 		?>
 		<input type="hidden" name="variable_post_id[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( $variation_id ); ?>" />
 		<input type="hidden" class="variation_menu_order" name="variation_menu_order[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( $variation_object->get_menu_order( 'edit' ) ); ?>" />
+		
+		<?php
+		/**
+		 * woocommerce_variation_header action.
+		 *
+		 * @since 2.5.0
+		 *
+		 * @param array   $variation_data
+		 * @param WP_Post $variation
```

Comment:
Missing description for `$variation`.

Reply:
Wasn't sure what to write here, I believe it's self-explanatory - `WP_Post $variation`.

Also, none of the other places in this file had a description for this param, even in the file header block.

Can you suggest a description, or should we leave it as is?

Reply:
For self-explanatory variables I've written just ".", to indicate that there's nothing more to say and it's not a missed comment for code quality tools.

Reply:
Resolved in a9afc72